openedx/core/djangoapps/catalog/utils.py

```diff
@@ -76,8 +76,9 @@ def get_programs(user=None, uuid=None, type=None):  # pylint: disable=redefined-
         return []
 
 
-def get_program_types(user=None):  # pylint: disable=redefined-builtin
-    """Retrieve all program types from the catalog service.
+def get_program_types(user=None, program_type_name=None):  # pylint: disable=redefined-builtin
```

Comment:
Do we still need this `program_type_name` filter? I don't see us using it anywhere at the moment.